DualMode-GRUBsetup.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+set -euo pipefail
+
+# Re-Install RPMs as necessary
+dnf -y reinstall grub2-pc
+
+# Move /boot/efi/EFI/redhat/grub.cfg as necessary
+[[ -e /boot/efi/EFI/redhat/grub.cfg ]] && \
```

Comment:
Won't this return 1 if the file _doesn't_ exist, and then errexit due to `set -e`? Is that the desired flow?

Reply:
It didn't fail when running, so, didn't trip over it. Lemme add suspenders…